src/argilla/labeling/text_classification/rule.py

```diff
@@ -143,6 +147,18 @@ def __call__(
             return self._label
 
 
+def add_rules(dataset: str, rules: List[LabelingRule]):
```

Comment:
The `LabelingRule` class is an internal data model used to communicate with the API. You should use the `Rule` data model and do the mappings to the `LabelingRule`, like function `load_rules` does.

```suggestion
def add_rules(dataset: str, rules: List[Rule]):
```